src/layouts/ReviewRequest/components/CancelRequestModal/CancelRequestModal.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalProps,
+  Text,
+} from "@chakra-ui/react"
+import { Button, ModalCloseButton } from "@opengovsg/design-system-react"
+
+export const CancelRequestModal = (props: ModalProps): JSX.Element => {
+  const { onClose } = props
+
+  return (
+    <Modal {...props}>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Text textStyle="h4" color="text.title.alt">
+            Cancel request?
+          </Text>
+        </ModalHeader>
+        <ModalCloseButton />
+
+        <ModalBody>
+          <Text textStyle="body-2">
+            The request to review will be cancelled but changes made will
+            remain.
+          </Text>
+        </ModalBody>
+
+        <ModalFooter>
+          <Button
+            textColor="text.body"
+            variant="clear"
+            mr="1rem"
+            onClick={onClose}
+          >
+            Go back
+          </Button>
```

Comment:
Should the colour scheme be secondary for this?

Also need to check with @NatMaeTan when hovering over this button, should it give a blue or purple background?

Reply:
i double checked with nat before, our `primary.500` might be wrong ._.

Reply:
Hmm I think we will have to adjust this in a separate PR.